docs/pages/config.rst

```diff
@@ -3,8 +3,6 @@
 Configuration
 =============
```

Comment:
I like the three examples for different ways to config with tokens. This is often a point of confusion, but with these improved SDK docs, this will be easier to point to, and expect users to read.